### ISSUE TITLE
Upload to dockerhub

### ISSUE THREAD

**Issue by AbdealiLoKo** [pywikibot-catfiles/docker-file-metadata#5]
@drtrigon I'm thinking it may be a wise idea to make multiple repos named `file-metadata-ubuntu` and so on. Then we can have tags like `file-metadata-ubuntu:14.04` for ubuntu versions.

Or do you think there should be only 1 docker repo named `file-metadata` and let users have tags like `file-metadata:ubuntu1404` to download the ubuntu image, and so on ?

**Comment by AbdealiLoKo** [comment]
This was about dockerhub repos, not github repos.
We can have Dockerfile.ubuntu, Dockerfile.centos, etc to associate to the different repos on dockerhub.

**Comment by drtrigon** [comment]
Actually I was up setting up the 2nd one; 1 repo, lots of tags. But any arguments are wellcome. If you wish we can even do both in parallel.
I was thinking we should have 1 main image (I vote for ubuntu) that we support and maintain. Is there a reson to have more? I have the impression code availability for experimenting is best on ubuntu. What do you think?
I was playing with the centos to see whether I can make ig work easily, not more. I would not insist on mainaining it... ;))

**Comment by drtrigon** [comment]
I would use `file-metadata:ubuntu-14.04` e.g. and `:latest` would be what we agree as default on.

**Comment by AbdealiLoKo** [comment]
The only reason to have multiple was because some people prefer centos more than Ubuntu. Plus, we can easily test for different OSes like that.

**Comment by drtrigon** [comment]
All on `doker-file-metadata`. If we can merge/pull `Dockerfile.centos` I can put it onto `file-metadata` too.

**Comment by AbdealiLoKo** [comment]
We don't need it on file-metadata

**Comment by AbdealiLoKo** [comment]
Uploading to dockerhub is not being done by Travis though right? Could we make another issue to do it with Travis? As it would test the image before pushing to docker

**Comment by AbdealiLoKo** [comment]
Nevermind, just saw https://github.com/pywikibot-catfiles/docker-file-metadata/issues/2
